src/components/HelloComponent.js

```diff
@@ -1,43 +1,66 @@
-import React, { Component } from 'react';
-import { connect } from 'react-redux';
-import PropTypes from 'prop-types';
-import customMessage from '../utils/customMessages';
-import firstMessage from '../actions/actions';
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import { MdRemove } from 'react-icons/md';
+import Card from './Card';
+import Footer from '../views/Footer';
 
-class HelloComponent extends Component {
-  constructor() {
-    super();
-    this.state = {};
-  }
-
-  render() {
-    const { message, displayMessage } = this.props;
-
-    return (
-      <div>
-        <h1>
-          {customMessage.welcomeToBarefootNomad}
-        </h1>
-        <h2>{message}</h2>
-        <button type="button" onClick={displayMessage}>
-          try redux
-        </button>
+const HelloComponent = () => (
```

Comment:
Why did you call this a HelloComponent? In my opinion, you should have renamed HomeComponent or LandingPageComponent or something in that way